DependencyInjection/DoctrineExtension.php

```diff
@@ -1123,4 +1132,32 @@ private function useMiddlewaresIfAvailable(
                 ->addTag('doctrine.middleware', ['connection' => $connName]);
         }
     }
+
+    /**
+     * TODO: remove this once we dropped support for ORM < 3.0
```

Comment:
```suggestion
     * TODO: remove this once we drop support for ORM < 2.14
```

Because 3rd argument is supported since 2.14. So once we start requiring it, then we don't need conditional logic and we can hardcode last argument.

Btw do we need to put it here instead of to `\Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler\DoctrineOrmMappingsPass::createXmlMappingDriver`?

Reply:
But with ORM 3 we don't need to hardcode anything, or? Then the validation is always enabled I believe. I mean then we can drop the whole method.